### ISSUE TITLE
[Bug] cloudCostReports helm value not working

### ISSUE THREAD

**Issue by dl-mai** [kubecost/cost-analyzer-helm-chart#3567]
### Kubecost Helm Chart Version

V2.3.4

### Kubernetes Version

v1.28.9

### Kubernetes Platform

Other (specify in description)

### Description

Rancher

### Steps to reproduce

Configure cloudCostReports not working
```
global:
  cloudCostReports:
    enabled: true
    reports:
      - title: "Example Cloud cost reports"
        window: "month"
        aggregateBy: "accountID"
        accumulate: "false"
        costMetric: "listCost"
        filters:
          - key: "provider"
            operator: ":"
            value: "AWS"
```

Also setting `Values.cloudCostReportConfigmapName`will result in a template error due to wrong indentation.

https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/cost-analyzer/templates/cost-analyzer-deployment-template.yaml#L783-L786

### Expected behavior

cloudCostReports configured via Helm should be displayed in reports

### Impact

Cannot use Cloud Reports with Custom Labels

### Troubleshooting

- [X] I have read and followed the [issue guidelines](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/ISSUE_GUIDELINES.md) and this is a bug impacting only the Helm chart.
- [X] I have searched other issues in this repository and mine is not recorded.

**Comment by kwombach12** [comment]
Hey @dl-mai. Thanks for logging this - we actually have a PR out to fix this which is undergoing QA now. We expect this to be resolved in our 2.4 release. 

[[Internal Ticket for Tracking](https://kubecost.atlassian.net/browse/ENG-2510)]

**Comment by chipzoller** [comment]
Internally noted as released in 2.3.5 so closing as complete.
